package.json

```diff
@@ -74,7 +74,7 @@
     "package.json": "npx sort-package-json"
   },
   "dependencies": {
-    "@rsksmart/rif-relay-client": "2.0.0-beta.6",
+    "@rsksmart/rif-relay-client": "github:rsksmart/rif-relay-client#fix/verify_with_relay_hub",
```

Comment:
We need to update it once we merge the relay-client PR

Reply:
updated